web/beacon-app/src/features/topics/components/TopicQueryResult/MetaDataTable.tsx

```diff
@@ -0,0 +1,32 @@
+import { t } from '@lingui/macro';
+import { Table } from '@rotational/beacon-core';
+import { ErrorBoundary } from '@sentry/react';
+
+const MetaDataTable = () => {
+  const initialColumns: any = [
+    { Header: t`Key`, accessor: 'key' },
```

Comment:
The table in the Figma link doesn't have a header, but I think that this may be the best way to work with the table component from the design system.